ui/app/AppLayouts/Chat/controls/community/ListDropdownContent.qml

```diff
@@ -13,19 +13,22 @@ StatusListView {
     id: root
 
     property var headerModel
+    property bool isHeaderVisible: true
+    property int maxHeight: 381 // default by design
 
-    signal headerItemClicked(int index)
-    signal itemClicked(string name, string shortName, url imageSource)
+    signal headerItemClicked(string key)
+    signal itemClicked(var key, string name, var shortName,  url iconSource, var subItems)
```

Comment:
why key and shortName are var ?

Reply:
Reason for `key` is answer in the next comment.
Reason for `shorName` is that `shortName` could not be mandatory for all models, so it can be a `string` or `undefined`. Here, for example, in case of token items, shorName is present, but in case of collectibles, no.